src/App.css

```
@@ -5,6 +5,10 @@
   text-align: center;
 }
 
+.new-logo {
```

This is a small change
```suggestion
.new-logo2 {
```